frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/simulated-regions-modal/simulated-regions-modal.component.html

```diff
@@ -8,35 +8,50 @@ <h4 class="modal-title">Simulierte Regionen</h4>
     ></button>
 </div>
 <div
-    class="modal-body"
+    class="modal-body p-0"
     style="height: min(80vh, 2000px); max-height: min(80vh, 2000px)"
 >
-    <div class="h-100 d-flex flex-column">
-        <nav
-            #nav="ngbNav"
-            ngbNav
-            [(activeId)]="currentSimulatedRegionId"
-            class="nav-tabs"
+    <div class="d-flex flex-row w-100 h-100">
+        <div
+            class="flex-shrink-1 mh-100 p-4 d-flex flex-column"
+            style="min-width: 20%"
         >
-            <ng-container
-                *ngFor="let id of simulatedRegionIds$ | async"
-                [ngbNavItem]="id"
+            <div class="flex-shrink-1">
+                <p>Example title</p>
```

Comment:
I would already specify the real title here, e.g. `<h5>Meldungen</h5>` (`h5` since the surrounding modal uses `h4` for its title).

We might also want to change the modal title and its button in the toolbar from *Simulierte Regionen* to something like *Simulations-Übersicht*, what do think about this?